src/test/java/uk/gov/pay/api/it/directdebit/pact/DirectDebitPaymentTest.java

```diff
@@ -109,14 +110,8 @@ private ValidatableResponse postPaymentResponse(String bearerToken, String paylo
                 .then();
     }
 
-    private static String paymentPayload(long amount, String returnUrl, String description, String reference, String email) {
-        return new JsonStringBuilder()
-                .add("amount", amount)
-                .add("reference", reference)
-                .add("email", email)
-                .add("description", description)
-                .add("return_url", returnUrl)
-                .build();
+    private String paymentLocationFor(String chargeId) {
```

Comment:
I guess this could be moved into your new Urls class? Then we can still remove the `extends PaymentResourceITestBase` and have this method available to the masses.

Reply:
hmm yeah I'll do that